src/app/components/course/CourseDetails/courseDetails.module.css

```diff
@@ -0,0 +1,347 @@
+.topBannerMainContainer {
+    background: linear-gradient(135deg, rgba(102, 0, 204, 0.5) 0%, rgba(74, 74, 255, 0.5) 100%), #E4DCFF;
+    padding: 3rem;
+    height: 635px;
+    max-width: 100%;
+    margin: auto auto;
+}
+
+.topBannerInstituteTitle {
+    display: flex;
+    align-items: center;
+    padding-left: 0;
+}
+
+.topBannerInstituteLogo {
+    width: 48px;
+    height: 48px;
+    border-radius: 50%;
+    object-fit: contain;
+    overflow: hidden;
+}
+
+.topBannerCourseTitle {
+    padding-left: 0;
+}
+
+.topBannerAdmissions {
+    background: var(--GhostWhiteSub);
+    height: auto;
+    border-radius: 8px;
+    margin-top: 7rem;
+    margin-left: 5rem;
+    padding: 16px;
+    text-align: center;
+    box-shadow: 0px 8px 20px rgba(0, 0, 0, 0.12);
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+}
+
+.topBannerAdmissionsBtnGroup {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+}
+
+.topBannerAdmissionsApplyNowBtn {
+    width: 135px;
+}
+
+.topBannerAdmissionsShortlistBtn {
+    width: 165px;
+}
+
+.middleBannerMainContainer {
+    background: var(--GhostWhiteSub);
+    width: 93%;
+    height: 220px;
+    border-radius: 16px;
+    box-shadow: 0px 8px 20px rgba(0, 0, 0, 0.12);
+    margin: -7rem auto 4rem auto;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+
+.middleBannerIconContainer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-weight: 500;
+    font-size: 22px;
+    gap: 0.6rem;
+}
+
+.middleBannerIconImageWrapper {
+    height: 64px;
+    background: var(--GhostWhiteVariant);
+    border-radius: 50%;
+    padding-bottom: 1rem;
+}
+
+.middleBannerIconImageWrapper img {
+    scale: 0.5;
+}
+
+.benefitTabContainer {
+    display: flex;
+    flex-direction: column;
+    width: 93%;
+    border-radius: 16px;
+    box-shadow: 0px 8px 20px rgba(0, 0, 0, 0.12);
+    margin: 0 auto 4rem auto;
+    background: var(--GhostWhiteSub);
+}
+
+.benefitTabContainerTitle {
+    height: 74px;
+    display: flex;
+    align-items: center;
+    padding-left: 24px;
+    background-color: var(--cherryPie);
+    border-top-left-radius: inherit;
+    border-top-right-radius: inherit;
+    margin-top: 0;
+    margin-bottom: 0;
+}
+
+.benefitTabContainerTabTitleGroup {
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    text-align: center;
+    height: 60px;
+    border-bottom: 1px solid var(--GhostWhiteVariant);
+}
+
+.benefitTabContainerTabTitleGroup button {
+    border: none;
+    background-color: transparent;
+    cursor: pointer;
+    width: 100%;
+    height: 100%;
+}
+
+.benefitTabContainerTabContentContainer {
+    border: none;
+}
```

Comment:
Its okay to make the selector name self-explanatory, but when it starts to get long, you can decide on the most important aspects to keep as part of the name, omit the others and shorten the name. For instance, `benefitTabContentContainer` is acceptable instead of `benefitTabContainerTabContentContainer`, this applies to other cases as well.